client/components/delete-modal.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+
+export default class DeleteModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookToDelete: {}
+    };
```

Comment:
I believe you can remove this state

Reply:
<img width="536" alt="Screen Shot 2022-09-22 at 11 55 43 AM" src="https://user-images.githubusercontent.com/106217355/191828912-0abb65bc-2bd0-4eff-80a5-a191e7514bf9.png">